.github/workflows/plugin-list.yml

```diff
@@ -0,0 +1,32 @@
+name: Update Plugin List
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  createPullRequest:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-verion: 3.8
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install packaging requests tabulate[widechars]
+      - name: Update Plugin List
+        run: python scripts/plugin_list.py
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
```

Comment:
The PR opened by this will target `master`, correct?

`master` only generates the documentation for https://docs.pytest.org/en/latest, while `stable` points to the latest tagged release, and by default users are directed to the `stable` version when typing `docs.pytest.org`.

Would it be possible to target the latest released branch somehow? Otherwise the updated list of plugins will only be available on `latest`, and the default location reached by users will be outdated in general.

Reply:
As described [here](https://github.com/marketplace/actions/create-pull-request#action-inputs):

base | Sets the pull request base branch. | Defaults to the branch checked out in the workflow.
-- | -- | --

This actually makes a lot of sense to me. We can also pass a ref to the checkout according to its [documentation](https://github.com/marketplace/actions/checkout). It says the ref can be tag (like stable) but i don't know how it would open PR from a detached HEAD. This seems to be a problem to me? Since the [output of commands can be used in the workflow](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#set-an-output-parameter-set-output) one could maybe come up with a way to get the correct branch based on the tag and pass it to the `ref` option of the checkout. I am not sure how this would work in regards to keeping the master branch up-to-date. Maybe a matrix of environment variables has to be created?

Reply:
> I am not sure how this would work in regards to keeping the master branch up-to-date.

If we can make it work by just targeting the latest stable, IMHO that's good enough.

Reply:
Maybe someone could help me out with this one. I do not know how to get the name of the branch with the latest release.  I don't think that checking out a tag would work. Also, i do not have the git repo structure set up to test this properly at [mcsitter/pytest-plugin-list](https://github.com/mcsitter/pytest-plugin-list). Just sorting by the tags on pytest currently spits out the dev version of version 6.2.0 as the latest tag . Should i just put `6.1.x` in there for the branch? That would mean that every minor version requires an update to the workflow.

Reply:
(Sorry I missed the notification for this one)

I think it is safe to filter every tag which doesn't match `\d+\.\d\.x`, so it will leave out the dev tags or any other tag which doesn't match our version scheme for branches.